examples/example_app/docs/requirements.txt

```diff
@@ -1,6 +1,5 @@
 Jinja2==3.1.3
 mkdocs==1.5.3
-mkdocs-include-markdown-plugin==4.0.4
-mkdocs-material==9.1.18
-mkdocstrings==0.22.0
-mkdocstrings-python==1.3.0
+mkdocs-material==9.5.8
+mkdocstrings==0.24.0
+mkdocstrings-python==1.8.0
```

Comment:
Since it has no functional impact to the end user, I don't think so? Could add it to the `.housekeeping` if you like though. Note that the changes to the `docs/requirements.txt` files really only impact ReadTheDocs though.